action.yml

```diff
@@ -9,6 +9,10 @@ inputs:
     description: Minimal version of python to be tested against.
     required: false
     default: "3.8"
+  max_python:
+    description: Maximal version of python to be tested against.
+    required: false
+    default: "3.12-dev"
```

Comment:
I still see a 3.12-dev as default when I do want to only see 3.12. We should not expose the "dev" aspect of 3.12 to the users.

Reply:
Whoops....changed it!